deploy/Provisioning-1674548289785.json

```diff
@@ -1553,8 +1654,8 @@
       {
         "current": {
           "selected": false,
-          "text": "crcs02ue1-prometheus",
-          "value": "crcs02ue1-prometheus"
+          "text": "crcp01ue1-prometheus",
+          "value": "crcp01ue1-prometheus"
```

Comment:
Ha this was not intentional, I made the changes on prod Grafana this needs to be fixed.